.gitignore

```diff
@@ -3,6 +3,11 @@
 .gradle/
 build/
 out/
+classes/
```

Comment:
IDE-specific stuff should go into your global gitignore instead of being per project. (I know, there's already IDE specific stuff in here, I want to remove that.)